autoscale/src/main/java/com/sequenceiq/periscope/controller/validation/AlertValidator.java

```diff
@@ -71,6 +71,17 @@ public void validateStopStartEntitlementAndDisableIfNotEntitled(Cluster cluster)
         }
     }
 
+    public boolean enableStopStartBasedOnEntitlement(Cluster cluster) {
```

Comment:
This code snippet is in the wrong place. This class is meant for validations ... and should not be changing actual configs.

Reply:
Shifting it to AutoScaleClusterCommonService.